web-client/src/app/views/manual-address/manual-address.page.html

```diff
@@ -0,0 +1,37 @@
+<ion-header class="ion-no-border">
+  <ion-toolbar>
+    <ion-buttons slot="start">
+      <ion-button (click)="dismiss(false)">
+        <ion-icon slot="icon-only" name="close-circle-outline"></ion-icon>
+      </ion-button>
+    </ion-buttons>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content class="lockscreen-page ion-text-center">
+  <ion-grid class="h-full">
+    <ion-row class="h-full ion-justify-content-center px-5">
+      <ion-col class="ion-align-self-center">
+        <ion-icon name="wallet-outline" class="!text-5xl"></ion-icon>
+        <h1 class="font-semibold font-nasalization">Wallet address</h1>
+        <ion-textarea
+          type="text"
+          placeholder="Enter recipient's wallet address"
+          [autofocus]="true"
+          class="p-0"
+          [(ngModel)]="address"
+        ></ion-textarea>
+
+        <ion-button
+          expand="block"
+          shape="round"
+          type="submit"
+          class="mt-4"
+          (click)="confirm()"
+          [disabled]="!address"
```

Comment:
Is is possible to implement validation / disabled for the wallet address here. See this function I used for the wallet access page: https://github.com/registreerocks/nautilus-wallet/blob/75bd6fd8be549242d905d4c7b0aa6492a298f391/web-client/src/app/views/wallet-access/wallet-access.page.ts#L39-L45

Not sure what the cleanest way would be to implement something like this, but some sort of validation here would help I think.

The `isValidAddress` is from the algosdk. See the imports.